tests/cli/workflow/rnafusion/conftest.py

```diff
@@ -226,7 +227,141 @@ def fixture_rnafusion_hermes_process(
 
 
 @pytest.fixture
-def mock_analysis_finish(rnafusion_dir: Path, rnafusion_case_id: str) -> None:
+def rnafusion_metrics_deliverables(rnafusion_case_id: str) -> dict:
+    return {
```

Comment:
I think this should be read from a fixture file instead of defined directly in the code. It is to big